testem.js

```diff
@@ -12,12 +11,14 @@ module.exports = {
     Chrome: {
       mode: 'ci',
       args: [
+        // --no-sandbox is needed when running Chrome inside a container
+        process.env.TRAVIS ? '--no-sandbox' : null,
```

Comment:
'--no-sandbox' should be always on

Reply:
I believe that's not the case with this public repo. Can you verify that?